### ISSUE TITLE
metric depth models

### ISSUE THREAD

**Issue by hcscysh** [DepthAnything/Depth-Anything-V2#200]
Hi,Young.In the scal question, you give a reply as follows.
Hi, our basic models produce the relative depth, without physical meaning of the value. However, our metric depth models produce the metric (absolute) depth, and the values are in meters.
I want to know the basic models if is depth_anything_v2_vitb.pth, and the metric depth models is depth_anything_v2_vitl.pth

**Comment by LiheYoung** [comment]
No, the basic relative-depth models are [list here](https://github.com/DepthAnything/Depth-Anything-V2#pre-trained-models), while the metric-depth models are [listed here](https://github.com/DepthAnything/Depth-Anything-V2/tree/main/metric_depth#pre-trained-models).
